dgraph/sample/schema.gql

```diff
@@ -1,20 +1,13 @@
-type Entity {
-  code
-  description
-  type
-  has_property
-  has_child
+type Entity @dgraph(type: "Entity") {
+    type: String! @dgraph(pred:"type") @search(by: [exact])
+    code: String @dgraph(pred:"code") @search(by: [exact])
+    description: String! @dgraph(pred:"description") @search(by: [term, exact, trigram])
```

Comment:
when querying - I don't find the `anyoftext` very helpful for what we need. Searching for `anyoftext: "Amox"` returns no results. Using the `fulltext` is more like what we need: `regexp: "/Amox/"` will return results.

```suggestion
    description: String! @dgraph(pred:"description") @search(by: [fulltext, exact])
```

Reply:
Yes, `anyoftext: "Amoxicillin"` returns results - that's not how mSupply is using the search atm though!

Reply:
Makes sense, seems like mSupply is doing more of a partial word match as entered in the UI? Sounds like a more useful search experience too 👍